source/docs/hardware/hardware-basics/can-wiring-basics.rst

```diff
@@ -0,0 +1,24 @@
+CAN Wiring Basics
+=================
+
+CAN is a two wire network that is designed to facilitate communication between multiple devices on your robot. It is recommended that CAN on your robot follow a “daisy-chain” topology. This means that the CAN wiring should usually start at your roboRIO and go into and out of each device successively until finally ending at the PDP.
+
+.. image:: images/can-wiring-basics/daisychain.png
+
+Standard Wiring
+---------------
+
+CAN is generally wired with yellow and green wire with yellow acting as the CAN-High and green as the CAN-Low signals.  Many devices show this yellow and green color scheme to indicate how the wires should be plugged in.
+
+CAN wiring from the roboRIO to the PCM.
+
+.. image:: ../../zero-to-robot/step-1/images/how-to-wire-a-robot/image15.jpg
+
+CAN wiring from the PCM to the PDP.
+
+.. image:: ../../zero-to-robot/step-1/images/how-to-wire-a-robot/image16.jpg
```

Comment:
I think I'd prefer using the absolute path over relative, since it's more flexible for renames.

```suggestion
.. image:: /docs/zero-to-robot/step-1/images/how-to-wire-a-robot/image16.jpg
```